vmaas/webapp/packages.py

```diff
@@ -78,11 +78,13 @@ def process_list(self, api_version, data):  # pylint: disable=unused-argument
 
             # If the package is third party, then remove it from result
             if not want_third_party and is_third_party:
-                del packagelist[pkg]
+                packagelist[pkg] = {}
+
+        if len(packagelist) == 1 and list(packagelist.values())[0] == {}:
```

Comment:
`last_change` shouldn't be removed. There should be no difference in response when single package or more packages are requested.